src/routes/guides/deployment-options/zerops.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Zerops
+order: 7
+mainNavExclude: true
+---
+
+import { TabsCodeBlocks } from "~/ui/tab-code-blocks";
+
+[Zerops](https://zerops.io) is a dev-first cloud platform that can be used to deploy both Static and SSR Solid Node.js Apps.
+
+For additional one-to-one support, details, and features, you can join the [Zerops Discord server](https://discord.gg/xxzmJSDKPT) and [visit the Zerops Docs](https://docs.zerops.io).
+
+Deploy and test Zerops Solid recipes with one click:
+- [Deploy Solid Node.js & Static Together](https://app.zerops.io/recipe/solidjs) - [Node.js](https://github.com/zeropsio/recipe-solidjs-nodejs) and [Static](https://github.com/zeropsio/recipe-solidjs-static).
+- [Deploy Solid Node.js](https://app.zerops.io/recipe/solidjs-nodejs) - [Source Repository](https://github.com/zeropsio/recipe-solidjs-nodejs)
+- [Deploy Solid Static](https://app.zerops.io/recipe/solidjs-static) - [Source Repository](https://github.com/zeropsio/recipe-solidjs-static)
+
+## Setting up an Account on Zerops
+
+1. Go to [Zerops Registration](https://app.zerops.io/registration) and sign up using GitHub, GitLab, or just your email.
+
+## Setting up your Project Infrastructure
+
+There are two ways to set up a Zerops project and a service:
+
+#### Using Project Add Wizard (GUI)
+
+1. Go to your [Zerops dashboard](https://app.zerops.io/dashboard/projects).
+2. Add a new project using your sidebar. If you're in compact mode, click on your profile and then "Add new project."
+3. You'll be redirected to a page where you can choose a service.
+
+##### For Static:
+1. Choose Static.
+2. Scroll down and change the hostname to your preference.
+3. Scroll down and click on the "Add New Static" button.
+
+##### For SSR - Node.js:
+1. Choose `Node.js` and select `version 20`.
+2. Scroll down and change the hostname to your preference.
+3. Scroll down and click on the "Add New Node.js" button.
+
+#### Using Project Import YAML
+
+**Note**: This is only used for project creation using YAML on the web interface—no need to add it to the project.
+
+1. Go to your [Zerops dashboard](https://app.zerops.io/dashboard/projects) and click on your profile icon if you are a new user. If not, check your sidebar and click on `Import Project`.
+
+##### Static:
+```yaml
+project:
+  name: recipe-solidjs
+
+services:
+  - hostname: app
```

Comment:
Maybe (optionally) explain that the hostname is not the actual hostname of your app, but a unique identifier inside your project, lest people try to replace it with their service's origin.

Reply:
Here (infrastructure import YAML) it means the actual hostname, it will be used for the both service name and the hostname under which the service will be accessible inside the project's internal network. 

It's the other place — `zerops.yml` (the file with build and start instructions that you place inside the repository) where `setup`s don't neccesarily have to match the hostname.

```yaml
zerops:
  - setup: app
    build: ...
    run:  ...
```

By default this would be the setup instruction for a service with hostname `app`, but there are edge cases (for example a monorepo with two completely different apps, belonging to different projects within which they are deployed to a service called `app`), in which case you can name the `setup`s whatever you desire and use param of the CLI deploy command / repository connection which setup should be used. 

But these are quite advanced concepts that I don't think are worth mentioning in Solid's docs.